bcipy/task/tests/core/test_handler.py

```diff
@@ -5,7 +5,7 @@
 from bcipy.task.control.criteria import CriteriaEvaluator, \
     MaxIterationsCriteria, MinIterationsCriteria, ProbThresholdCriteria
 from bcipy.task.control.query import NBestStimuliAgent
-from bcipy.helpers.task import alphabet
+from bcipy.language.main import alphabet
```

Comment:
Why re-import this from language.main rather than from where it's defined in helpers.task?